packages/react-divider/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "9.0.0-alpha.10",
```

Comment:
Is there a reason for starting this at 10 not 0? Doesn't matter much but just curious.

Reply:
Probably just an oversight, we should start at 0.

Reply:
Updated